certbot-postfix/certbot_postfix/constants.py

```diff
@@ -0,0 +1,42 @@
+"""Postfix plugin constants."""
+
+POLICY_FILENAME = "starttls_everywhere_policy"
+
+CA_CERTS_PATH = "/etc/ssl/certs/"
+
+MINIMUM_VERSION = (2, 6,)
+
+# If the value of a default VAR is a tuple, then the values which
+# come LATER in the tuple are more strict/more secure.
+# Certbot will default to the first value in the tuple, but will
+# not override "more secure" settings.
+
+ACCEPTABLE_SECURITY_LEVELS = ("may", "encrypt")
+ACCEPTABLE_CIPHER_LEVELS = ("medium", "high")
+
+TLS_VERSIONS = ("SSLv2", "SSLv3", "TLSv1", "TLSv1.1", "TLSv1.2")
+# Should NOT use SSLv2/3.
+ACCEPTABLE_TLS_VERSIONS = ("TLSv1", "TLSv1.1", "TLSv1.2")
+
+# Default variables for a secure MTA server [receiver].
+DEFAULT_SERVER_VARS = {
+    "smtpd_tls_mandatory_protocols": "!SSLv2, !SSLv3",
+    "smtpd_tls_protocols": "!SSLv2, !SSLv3",
+    "smtpd_tls_security_level": ACCEPTABLE_SECURITY_LEVELS,
+    "smtpd_tls_ciphers": ACCEPTABLE_CIPHER_LEVELS,
+    "smtpd_tls_eecdh_grade": "strong",
```

Comment:
Yep, it is. Thanks for the catch! I noted this at some point, but forgot to follow up on it.

Reply:
Thanks for verifying it is an issue.

Without further research, I'm not positive what size param we should use (although I suspect it's 2048 bit) and I'm not sure the format the file needs to be in for Postfix, but the code I mentioned in my initial comment is in the `certbot.common.Installer` class in case it's useful.

Reply:
Whoops. Correct path is `certbot.plugins.common.Installer`. The Postfix Installer in this PR already inherits from this class.